legendary/cli.py

```diff
@@ -1287,7 +1287,9 @@ def import_game(self, args):
 
         # get everything needed for import from core, then run additional checks.
         manifest, igame = self.core.import_game(game, args.app_path, platform=args.platform)
-        exe_path = os.path.join(args.app_path, manifest.meta.launch_exe.lstrip('/'))
+        exe_path = os.path.join(args.app_path, manifest.meta.launch_exe.lstrip('/').replace('/', os.sep))
```

Comment:
Why bother replacing the separator with the OS-specific one? Windows supports both, Linux will only ever use `/`.

Reply:
*I think* at the time I wasn't aware that this could only ever be `/` on non-Windows